capabilityreceiver/README.md

```diff
@@ -0,0 +1,60 @@
+### Capability Receiver
```

Comment:
I compared this read me file to all the read me files already existing for the other individual services, and saw that this read me was set up slightly differently. In the existing files, the sections are ordered Configuration, Test Configuration, and then Building and running, where in this read me Building and running seems to come first and does not have the same header titles. Just thought I would point it out in case you want all the formats to be the same across all the individual services read me files!